spring-components/rdf4j-spring/src/main/java/org/eclipse/rdf4j/spring/dao/exception/Rdf4JSpringException.java

```diff
@@ -0,0 +1,26 @@
+package org.eclipse.rdf4j.spring.dao.exception;
+
+public class Rdf4JSpringException extends RuntimeException {
```

Comment:
Can we make this extend RDF4JException (which in turn extends RuntimeException)?

Reply:
ok

Reply:
done (also for all other exceptions)